src/app/shared/components/sidemenu/sidemenu.component.ts

```
@@ -195,27 +195,28 @@ export class SidemenuComponent implements OnInit {
 
-  getTeamOptions(): Partial<SidemenuItem>[] {
+  getTeamOptions(isOnboardingPending: boolean): Partial<SidemenuItem>[] {
     const showTeamReportsPage = this.primaryOrg?.id === (this.activeOrg as Org)?.id;
 
     const { allowedReportsActions, allowedAdvancesActions } = this.allowedActions;
     const teamOptions = [
       {
         title: 'Team expense reports',
-        isVisible: allowedReportsActions && allowedReportsActions.approve && showTeamReportsPage,
+        isVisible:
+          allowedReportsActions && allowedReportsActions.approve && showTeamReportsPage && !isOnboardingPending,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)